packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -74,6 +74,8 @@ const renderTrigger = ({ t, size, selectedOptions, onRemove, renderTag, tagLabel
 
   return (
     <sc.Trigger
+      aria-haspopup="true"
+      aria-expanded={isOpen}
```

Comment:
Do you think we should also have `aria-controls` here?
https://www.w3.org/TR/wai-aria-1.1/#aria-controls

Reply:
We do use it here:
https://github.com/acl-services/paprika/blob/master/packages/OverflowMenu/src/components/Trigger/Trigger.js#L20

Reply:
Yeah, lets add it